openrtb_ext/bidders.go

```diff
@@ -23,28 +23,42 @@ const BidderNameGeneral = BidderName("general")
 // BidderNameContext is reserved for first party data.
 const BidderNameContext = BidderName("context")
 
-// These names _must_ coincide with the bidder code in Prebid.js, if an adapter also exists in that project.
-// Please keep these (and the BidderMap) alphabetized to minimize merge conflicts among adapter submissions.
-// The bidder name 'general' is not allowed since it has special meaning in message maps.
+func (name BidderName) MarshalJSON() ([]byte, error) {
```

Comment:
Why do we need these methods? Does this `MarshalJSON()` do anything different from the default?

Reply:
Not sure. This was moved up from further down in the file. I just wanted to keep the `BidderName` receiver funcs together.